nisse/services/xlsx_document_service.py

```diff
@@ -27,7 +27,8 @@ def save_report(file_path, date_from, date_to, time_entries, project_name):
         sheet.column_dimensions['A'].width = 20
```

Comment:
one more minor fix is needed - can we have rows sorted by date in the spreadsheet... ?